packages/x-date-pickers/src/hooks/usePickerActionsContext.ts

```diff
@@ -0,0 +1,20 @@
+'use client';
+import * as React from 'react';
+import { PickerActionsContext } from '../internals/components/PickerProvider';
+
+/**
+ * Returns a sub-set of the context passed by the picker that wraps the current component.
+ * If only contain the actions and never causes a re-render of the component using it.
+ */
+export const usePickerActionsContext = () => {
```

Comment:
I created this hook because, `usePickerContext` contains things that update quite frequently and some component only need actions that don't trigger a re-render.

Let me know if you think it's too soon for this kind of optimizations.

Reply:
no, IMO its a great thing to have